CondCore/ESSources/plugins/CondDBESSource.cc

```diff
@@ -759,6 +759,7 @@ void CondDBESSource::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   dbParams.addUntracked<std::string>("security", "");
   dbParams.addUntracked<int>("messageLevel", 0);
   dbParams.addUntracked<int>("connectionTimeout", 0);
+  dbParams.addObsoleteUntracked<std::string>("transactionId", "")->setComment("This parameter is not strictly needed by PoolDBESSource, but the WMCore infrastructure requires it. Candidate for deletion");
```

Comment:
sorry it looks like you can't pass a default to `addObsoleteUntracked`, see:

https://github.com/cms-sw/cmssw/blob/6cad17bc389ac53d9b2193d99f5a76713a7a99f4/FWCore/ParameterSet/interface/ParameterSetDescription.h#L151

(which kind of makes sense :) )

Reply:
So this:
```suggestion
  dbParams.addObsoleteUntracked<std::string>("transactionId")->setComment("This parameter is not strictly needed by PoolDBESSource, but the WMCore infrastructure requires it. Candidate for deletion");
```
should work right?

Reply:
I would hope so.